content/docs/command-reference/repro.md

```diff
@@ -124,12 +127,11 @@ only execute the final stage.
 - `-i`, `--interactive` - ask for confirmation before reproducing each stage.
   The stage is only executed if the user types "y".
 
-- `-p`, `--pipeline` - reproduce the entire pipelines that the stage file
-  `targets` belong to. Use `dvc dag <target>` to show the parent pipeline of a
-  target stage.
+- `-p`, `--pipeline` - reproduce the entire pipelines that the `targets` belong
+  to. Use `dvc dag <target>` to show the parent pipeline of a target.
 
-- `-P`, `--all-pipelines` - reproduce all pipelines, for all the stage files
-  present in `DVC` repository.
+- `-P`, `--all-pipelines` - reproduce all pipelines, for all the stages present
```

Comment:
does it reproduce all pipelines within one dvc.yaml or within all of them? (search recusrsively)?

Reply:
It reproduces all of the pipelines (recursive search).

Reply:
I'm assuming all pipelines in all dvc.yaml files but not sure. Let's test this @sarthakforwet ? Keep in mind you can have one dvc.yaml in each subdirectory of the project. Let's update in #1615 if needed. Thanks

Reply:
> Let's test this @sarthakforwet ?

Already tested it @jorgeorpinel. Here's the results - 

![image](https://user-images.githubusercontent.com/46163196/88328633-5ffc5a80-cd46-11ea-9e0d-c8d99e11ae69.png)

Directory Structure - 

![image](https://user-images.githubusercontent.com/46163196/88328894-bb2e4d00-cd46-11ea-9598-9c01563b2666.png)

Reply:
Just tested. -P makes it recursive indeed (checks all dvc.yaml files). I rewrote the description anyway since it didn't make much sense. See #1615.

Reply:
The funny think is that `status` is recursive by default, but `repro` needs `-P`... Is that a bug? Cc @efiop

Reply:
thanks @sarthakforwet, for testing this. I would make the note more explicit then

Reply:
I already updated the note.

Reply:
> status is recursive by default, but repro needs -P

Extacted this matter iterative/dvc/issues/4273 with much more detail. PTAL